crates/node/src/cli/rpc.rs

```diff
@@ -185,6 +189,16 @@ impl RpcParams {
         SocketAddr::new(listen_addr.into(), self.rpc_port)
     }
 
+    pub fn addr_admin(&self) -> SocketAddr {
+        let listen_addr = if self.rpc_external && self.rpc_endpoints == RpcEndpoints::Unsafe {
```

Comment:
It seems unusual to require the user to enable RpcEndpoints::Unsafe to expose admin RPCs under Docker

Reply:
To me this seems fine when considering a lot of existing Madara options can be quite confusing as is. Something worth investigating in the future would be configuration presets for node operators and docker for example, so calling somehting like `madara --preset docker` vs `madara --preset operator` will automatically set this up. I see several issues in the way in how clap works that might make this quite complicated though.